sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/caches/RxCollectionCache.java

```diff
@@ -29,9 +31,18 @@
     private final AsyncCache<String, DocumentCollection> collectionInfoByNameCache;
     private final AsyncCache<String, DocumentCollection> collectionInfoByIdCache;
 
+    public static void serialize(CosmosClientState state, RxCollectionCache cache) throws IOException {
+        state.serializeCollectionInfoByIdCache(cache.collectionInfoByIdCache);
```

Comment:
Are routing addresses also cached?

Reply:
Saw this as TODO in description. Thanks.

---
In reply to: [545885944](https://github.com/Azure/azure-sdk-for-java/pull/18184#discussion_r545885944) [](ancestors = 545885944)

Reply:
This PR provides the foundation for the cache broadcast but for now I only wired up the CollectionCache serialization.

Please see PR description TODOs for more info.